tests/manage/test_verify_rbd_based_rwo_pvc.py

```diff
@@ -0,0 +1,210 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.resources.pod import run_io_and_verify_mount_point
+from ocs_ci.ocs.resources.pvc import PVC
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.utility import templating
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.exceptions import (
+    TimeoutExpiredError, CommandFailed, UnexpectedBehaviour
+)
+from tests import helpers
+from tests.fixtures import create_ceph_block_pool, create_rbd_secret
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def test_fixture(request):
+    """
+    Setup and teardown
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
+    """
+    Create new pod using available PVC
+    """
+    # Create Storage Class with reclaimPolicy: Delete
+    self.sc_obj = helpers.create_storage_class(
+        interface_type=constants.CEPHBLOCKPOOL,
+        interface_name=self.cbp_obj.name,
+        secret_name=self.secret_obj.name,
+        reclaim_policy='Delete'
+    )
+
+    # Create PVC with 'accessModes' 'ReadWriteOnce'
+    pvc_data = templating.load_yaml_to_dict(constants.CSI_PVC_YAML)
```

Comment:
Please use helpers.create_pvc()

Reply:
helpers.create_pvc() is not used here because 'accessModes' has to be set for pvc.